<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageImp</class>
 <widget class="QWidget" name="PageImp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QStackedWidget" name="WindgetSetImp">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="WidgetImp">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <item>
        <widget class="QTableWidget" name="TabParams">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AllEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>选中</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端一</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端二</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电压</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>次数</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电晕</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>相位</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>面积</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>差积</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>波形</string>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="LayoutCtrl" columnstretch="1,0,0,0,0,0,0">
         <item row="0" column="1">
          <spacer name="SpacerCtrl">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="BtnExitImp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>保存退出</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="BtnAvarage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>添加样品</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="BtnExit">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="BoxStation">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>左工位</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>右工位</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="BtnSampleImp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>自动采集</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="BoxOffset">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>取消电压反馈</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WidgetWave">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
       <item>
        <widget class="QGroupBox" name="BoxWave">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Waveform" name="LabelWave">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="LayoutCtrl_2">
         <item row="0" column="4">
          <widget class="QPushButton" name="BtnFrequcy1">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>拉伸波形</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="BtnExitWave">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="TextStart">
           <property name="text">
            <string>计算区间起点:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="BoxStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximum">
            <number>199</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="TextEnd">
           <property name="text">
            <string>计算区间终点:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="BoxEnd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="minimum">
            <number>200</number>
           </property>
           <property name="maximum">
            <number>399</number>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="BtnFrequcy3">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>压缩波形</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="BtnFrequcy2">
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="text">
            <string>重新采样</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Waveform</class>
   <extends>QLabel</extends>
   <header>Waveform.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
